tools/rollup-bundle.js

```diff
@@ -0,0 +1,50 @@
+var _ = require('lodash');
+
+var rollup = require('rollup');
+var rollupAlias = require('rollup-plugin-alias');
+var rollupInject = require('rollup-plugin-inject');
+var rollupNodeResolve = require('rollup-plugin-node-resolve');
+
+var fs = require('fs');
+var tslib = require('tslib');
+var path = require('path');
+
+module.exports = function rollupBundle(options) {
+  var dest = options.dest;
+  var sourcemapFullFile = dest + '.map';
+
+  rollup.rollup({
+    input: options.input,
+    plugins: [
+      rollupAlias(options.aliases),
+      rollupNodeResolve({
+        jsnext: true,
+      }),
+      rollupInject({
+        exclude: 'node_modules/**',
+        modules: _.mapValues(tslib, function (value, key) {
+          return ['tslib', key];
+        }),
+      }),
+    ],
+  }).then(function (bundle) {
+    return bundle.generate({
+      format: 'umd',
+      name: 'rxjs',
+      amd: {
+        id: 'rxjs'
```

Comment:
this is the only line I recognize as being load-bearing. Why is the rest of the change so large? is there some refactoring going on at the same time?

Reply:
As far as I remember, the current rollup version used by RXJS didn’t support AMD ids. 

Since then Rollup API has changed, which caused a change in the build, and preferred the extract it into a common function.

Re the bazel files, Greg informed me that we should probably remove these files.

Reply:
I see - depending on what the rxjs maintainers prefer, it might be easier to land in two PRs: update rollup version, and separately change the id and remove Bazel files